packages/angular/package.json

```diff
@@ -21,70 +21,71 @@
     "_install": "npm install",
     "ng": "ng",
     "start": "npm run develop:package",
-    "_build": "ng build --prod",
+    "_build": "ng build --configuration production",
```

Comment:
Is the configuration part needed since the default is production?
Also, were both the configurations tested?